src/test/scala-2.13/org/apache/james/gatling/simulation/Configuration.scala

```diff
@@ -23,7 +23,12 @@ object Configuration {
   val WEBADMIN_PROTOCOL = Properties.envOrElse("WEBADMIN_PROTOCOL", "http")
   val BaseJamesWebAdministrationUrl = new URI(s"$WEBADMIN_PROTOCOL://$WebadminServerHostName:$WEBADMIN_PORT").toURL
 
-  val DURATION_PROPERTY = Properties.envOrNone("DURATION") match {
+  val SCENARIO_DURATION_PROPERTY = Properties.envOrNone("SCENARIO_DURATION") match {
+    case Some(duration) => Some(duration.toInt minutes)
+    case _ => None
+  }
+
+  val INJECTION_DURATION_PROPERTY = Properties.envOrNone("INJECTION_DURATION") match {
```

Comment:
We can avoid a breaking change by making both SCENARIO_DURATION and INJECTION_DURATION default to DURATION if unspecified.